package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "contexture-react",
-  "version": "1.38.6",
+  "version": "1.38.7",
```

Comment:
this should be a minor version since it's adding a new feature and isn't a bugfix